shared/validate/validate.go

```diff
@@ -755,6 +757,34 @@ func IsListenAddress(allowDNS bool, allowWildcard bool, requirePort bool) func(v
 	}
 }
 
+// IsListenAddressList returns a validator for a set of comma-separated addresses.
+func IsListenAddressList(allowDNS bool, allowWildcard bool, requirePort bool) func(value string) error {
```

Comment:
I think you can probably use 

https://github.com/lxc/lxd/blob/master/shared/validate/validate.go#L181

here by passing an intermediate anonymous function to it that calls IsListenAddress with the arguments you need.

Reply:
Oh thanks for that, I was looking at `IsNetworkAddressList` which already exists.

Reply:
Yeah we should probably get rid of that too.